<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAccueil</class>
 <widget class="QWidget" name="PageAccueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="topMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-50</y>
          <width>1091</width>
          <height>635</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>52</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>50</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>EasyTicket</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="horizontalSpacing">
            <number>40</number>
           </property>
           <property name="verticalSpacing">
            <number>60</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>60</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Identifiant/e-mail :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Mot de passe :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="boutonConnexion">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>19</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Connexion</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="boutonCreationCompte">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Cr√©er un compte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
